src/dataflow/components/dataflow-program.tsx

```diff
@@ -220,6 +223,36 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
   private setProgramRunTime = (time: number) => {
     this.setState({programRunTime: time});
   }
+  private generateProgramData = () => {
+    let programName: any = "my-program";
```

Comment:
I guess we should really make a specific type for a program that is going to be run on the server to remove the "any"

Reply:
Good point.